src/lib.rs

```diff
@@ -35,6 +35,20 @@ use colored::*;
 use log::{Level, LevelFilter, Log, Metadata, Record, SetLoggerError};
 use std::collections::HashMap;
 
+macro_rules! LOGGING_FMT {
+    () => {
+        "{ts}{lvl}{tgt}{thr}{msg}"
```

Comment:
I can do that, actually i had that first, but it felt a bit like a eyesore. `print!("{timestamp}", timestamp = timestamp)`. Kindof code duplication. IIRC for future Rust versions some improvements which make soemthing like `print!("{timestamp}", timestamp)` are under discussion. When that becomes standard then I am all in.

Reply:
I'm strongly for clear code over pretty code here - it might be more verbose but it makes it a lot easier to avoid things getting mixed up when working on the code.